src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('container');
+
+  container.classList.add('notification');
+
+  if (type === 'success') {
+    container.classList.add('notification', 'success');
+  } else if (type === 'error') {
+    container.classList.add('notification', 'error');
+  } else if (type === 'warning') {
+    container.classList.add('notification', 'warning');
```

Comment:
The class 'notification' is being added twice in each condition. You only need to add it once before the conditional statements.